src/Graph.cc

```diff
@@ -938,7 +940,7 @@ bool sea_dsa::Graph::computeCalleeCallerMapping(
       Cell &nc = callerG.mkCell(*kv.first, Cell());
       if (!simMap.insert(c, nc)) {
         if (reportIfSanityCheckFailed) {
-          errs() << "ERROR 1: callee is not simulated by caller at "
```

Comment:
seems out of sync with the main branch -- you can fix that by rebasing to stay on top of it

Reply:
@nhamlv-55 : also run `clang-format` and split into several commits based on what is being added in the commit

Reply:
https://stackoverflow.com/questions/6217156/break-a-previous-commit-into-multiple-commits